src/CommandingOfficers/Tech.java

```diff
@@ -257,7 +260,48 @@ public GameEventQueue getEvents(MapMaster gameMap)
       return abilityEvents;
     }
 
+    @Override
+    public Collection<DamagePopup> getDamagePopups(GameMap gameMap)
+    {
+      ArrayList<DamagePopup> output = new ArrayList<DamagePopup>();
+
+      boolean log = true;
+      Set<XYCoord> dropLocs = new HashSet<XYCoord>();
+      for( int i = 0; i < numDrops; ++i )
+      {
+        XYCoord techDrop = findDropLocation(gameMap, dropLocs, log);
```

Comment:
The projected drops don't always match the actual drops even with full visibility, I think because `findDropLocation` isn't deterministic in how it chooses spaces with equal scores.
Also, the "Drop" popups don't really look like they are in the right spot when there's nobody there to squash. I might add like, a little down arrow when there's no resident.

Reply:
What kind of down arrow?

Reply:
As a quick mock-up, maybe something like this:
![Image20000](https://user-images.githubusercontent.com/16439262/92988373-86aa5800-f490-11ea-9bac-cd5bf5b1a9ce.png)

Reply:
Drop arrows added. You want to make a sub-PR or something to make Tech drops deterministic?
I will say that Sturm actually has an RNG component on SCOP... but he only drops one thing, so I could just give all 3 options if I implemented that.

Reply:
I could probably do that. I had an idea I never implemented that would effectively do that, I think.

How does Sturm's RNG work?

Reply:
A-ha! I think I now know why I accidentally ended up editing a post of yours. It doesn't actually *show* the quote option until you've initiated a comment.

> I could probably do that. I had an idea I never implemented that would effectively do that, I think.

Hokay. I have another WIP branch that touches the same general area though. Should maybe chat on how to deal with that.
> How does Sturm's RNG work?

Kinda like Rachel, he has like 3 heuristics for picking his drop location that he randomizes between. It's otherwise deterministic.

Reply:
I'm playing with fonts right now. I guess just do your thing, and I probably won't get to it until you're done anyway.

Reply:
Made an issue for it.